### PR TITLE
Add "How to make a payment" user guide

### PR DESCRIPTION
**Pull request by rating89us**
Credits to @aantonop (https://mobile.twitter.com/aantonop/status/1333500376461393920) for the explanation on section "What should I do if my transaction is not confirmed or has failed?"

### CODE REVIEW

**Comment by rating89us** [comment]
@erciccione Any idea why the `@blockchain` moneropedia term is not loading its link on the preview?

**Comment by erciccione** [comment]
You have to add `{% t global.lang_tag %}` to `resources/user-guides/make-payment.md ` to make the ruby plugin work for this guide.

**Comment by rating89us** [comment]
Ready for review

File: _i18n/ar/resources/user-guides/make-payment.md
Comment (erciccione) [inline _i18n/ar/resources/user-guides/make-payment.md:3]: This title is not needed.

File: _i18n/ar/resources/user-guides/make-payment.md
Comment (erciccione) [inline _i18n/ar/resources/user-guides/make-payment.md:25]: @openalias, we have a Moneropedia entry for it.

File: _i18n/ar/resources/user-guides/make-payment.md
Comment (erciccione) [inline _i18n/ar/resources/user-guides/make-payment.md:69]: Add a newline between these two lines.

File: _i18n/ar/resources/user-guides/make-payment.md
Comment (erciccione) [inline _i18n/ar/resources/user-guides/make-payment.md:63]: Add newline after this header.

File: _i18n/ar/resources/user-guides/make-payment.md
Comment (erciccione) [inline _i18n/ar/resources/user-guides/make-payment.md:79]: Add a newline after this header.

File: _i18n/ar/resources/user-guides/make-payment.md
Comment (erciccione) [inline _i18n/ar/resources/user-guides/make-payment.md:77]: Better use something else instead of `Monero IRC channels`, since matrix channels are listed as well on that page.

**Review by erciccione — CHANGES_REQUESTED** [review]
Some changes to be applied to all guides.

File: _i18n/ar/resources/user-guides/make-payment.md
Comment (SamsungGalaxyPlayer) [reply _i18n/ar/resources/user-guides/make-payment.md:77]: "Monero public chat rooms"?

File: _i18n/ar/resources/user-guides/make-payment.md
Comment (SamsungGalaxyPlayer) [inline _i18n/ar/resources/user-guides/make-payment.md:15]: Add "This status can be seen in the lower left orange bars of the GUI."

File: _i18n/ar/resources/user-guides/make-payment.md
Comment (SamsungGalaxyPlayer) [reply _i18n/ar/resources/user-guides/make-payment.md:25]: We should use donate.getmonero.org to match the contributing page, here and below.

File: _i18n/ar/resources/user-guides/make-payment.md
Comment (rating89us) [reply _i18n/ar/resources/user-guides/make-payment.md:69]: Why?

Comment (erciccione) [reply _i18n/ar/resources/user-guides/make-payment.md:68]: po4a won't parse it correctly because they are two different markdown entities.

**Review by erciccione — APPROVED** [review]

**Comment by HoverHalver** [comment]
I did read this user guide, and imho it is quite well done.
Thanks @rating89us.

**Comment by rating89us** [comment]
added table of contents and more information about transaction fee

**Review by erciccione — APPROVED** [review]
The content is ok. Personally, i would prefer a shorter guide, but depending by who it is directed to, it can be ok.

Let's wait for more reviews/feedback before moving it to 'ready to merge'

**Comment by rating89us** [comment]
fixed merge conflict

**Review by erciccione — APPROVED** [review]

**Pull request merged**
